src/envelopeCommands.cpp

```diff
@@ -657,3 +657,40 @@ void postSelectMultipleEnvelopePoints(int command) {
 		translate_plural("{} point selected", "{} points selected", count),
 		count));
 }
+
+void cmdMoveSelEnvelopePoints(Command* command) {
+	TrackEnvelope* envelope;
+	double offset{0.0};
```

Comment:
nit: Out of curiosity, why did you choose to 0 initialise offset here but not envelope (nullptr)? Neither of these needs to be initialised because they are both immediately initialised below, but I can understand playing it safe... if that were consistent.

Anyway, please initialise both or don't initialise both. I'm happy either way. :)